dev/oracle/latest/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved
+
+services:
+  oraclexedb:
+    container_name: oraclexedb
+    image: gvenzl/oracle-free:23.4-slim
+    environment:
+      ORACLE_PASSWORD: password
+      ORACLE_DATABASE: XEPDB1
```

Comment:
@hjamil-24 Should this be freepdb1 now as we do not have xepdb in Oracle Database 23ai?

Reply:
This is fine. Its just naming the PDB to XEPDB1. This is done to keep the name of PDB same as was in version 6 and `oracle-oldest`.

Reply:
Ok. Thanks